src/arch/xtensa/CMakeLists.txt

```diff
@@ -369,24 +369,24 @@ if(NOT DEFINED RIMAGE_IMR_TYPE)
 endif()
 
 if(MEU_PATH OR DEFINED MEU_NO_SIGN)
-	if(NOT DEFINED MEU_OFFSET)
-		execute_process(
-			COMMAND ${MEU_PATH}/meu -ver
-			OUTPUT_VARIABLE MEU_VERSION_FULL_TEXT
-			OUTPUT_STRIP_TRAILING_WHITESPACE
-		)
-
-		string(REGEX MATCH "Version:[\t\n ]*([^\t\n ]+)" ignored "${MEU_VERSION_FULL_TEXT}")
-		set(MEU_VERSION ${CMAKE_MATCH_1})
-
-		if(MEU_VERSION VERSION_LESS 12.0.0.1035)
-			set(MEU_OFFSET 1152)
-		elseif(MEU_VERSION VERSION_LESS 15.0.0.7041)
-			set(MEU_OFFSET 1088)
-		else()
+#	if(NOT DEFINED MEU_OFFSET)
+#		execute_process(
+#			COMMAND ${MEU_PATH}/meu -ver
+#			OUTPUT_VARIABLE MEU_VERSION_FULL_TEXT
+#			OUTPUT_STRIP_TRAILING_WHITESPACE
+#		)
+
+#		string(REGEX MATCH "Version:[\t\n ]*([^\t\n ]+)" ignored "${MEU_VERSION_FULL_TEXT}")
+#		set(MEU_VERSION ${CMAKE_MATCH_1})
+
+#		if(MEU_VERSION VERSION_LESS 12.0.0.1035)
+#			set(MEU_OFFSET 1152)
+#		elseif(MEU_VERSION VERSION_LESS 15.0.0.7041)
+#			set(MEU_OFFSET 1088)
+#		else()
 			set(MEU_OFFSET 1344)
-		endif()
-	endif()
+#		endif()
```

Comment:
can you please remove these changes. These are for testing purpose only

Reply:
Right!!! Shall remove it.